docs/how-to-write-conventions/README.md

```diff
@@ -183,6 +187,27 @@ to that system. For example, when adding a new `db.system.name` value, create
 documentation and span definitions that show how generic attributes work for that
 database.
 
+### Defining entities
+
+Follow the [Entity Modeling Guide](resource-and-entities.md).
+
+### Defining Telemetry Signals
+
+The first step in defining a telemetry signal is to decide on what type of signal to use based on their strengths and weaknesses,
+as well as what you want to achieve.
+
+Here are some rules to guide you in choosing the correct signal type:
+
+- I want to debug/investigate the operations which are being performed including hoe long. Use a span.
+- I want to see what operations are triggered from another operation. Use a span.
+- I want to analyze how long operations take to complete. Use a metric.
+- I want to see how much/many of a resource is used/available. Use a metric.
```

Comment:
Apologies if I plug the "SRE" mindset here, but this would be called "saturation" golden signals.
It's possible we could just have a rule of:

```
- I want to track golden signals (saturation, errors, latency, volume).  Use a metric.
```